Server/mods/deathmatch/logic/CConsole.cpp

```diff
@@ -119,10 +119,10 @@ bool CConsole::HandleInput(const char* szCommand, CClient* pClient, CClient* pEc
     return false;
 }
 
-void CConsole::AddCommand(FCommandHandler* pHandler, const char* szCommand, bool bRestricted)
+void CConsole::AddCommand(FCommandHandler* pHandler, const char* szCommand, bool bRestricted, string sConsoleHelp)
 {
     // Make a command class and add it to the list
-    CConsoleCommand* pCommand = new CConsoleCommand(pHandler, szCommand, bRestricted);
+    CConsoleCommand* pCommand = new CConsoleCommand(pHandler, szCommand, bRestricted, sConsoleHelp);
     m_Commands.push_back(pCommand);
```

Comment:
imho keep using `const char*`, not need for `string` here at all. (if you really want to use one, use `std::move`). Also, wrong hungarian notation, it should be `str` not `s`